deploy/aws/README.md

```diff
@@ -22,7 +22,7 @@ export KEYNAME=YOUR_KEY # Name of an existing EC2 KeyPair to enable SSH access t
 export SUBNET=YOUR_AWS_SUBNET # Subnet that EC2 instances will be created in. Can be found from AWS console or running command: aws ec2 describe-subnets
```

Comment:
Additional note: add which docker image's container to look for from `docker ps` command and note that it might take some time to download